feature/ledgermanual/src/main/java/com/moneymong/moneymong/ledgermanual/view/LedgerManualCategoryBottomSheet.kt

```diff
@@ -0,0 +1,301 @@
+
+    MDSBottomSheet(
+        modifier = modifier,
+        sheetState = sheetState,
+        onDismissRequest = onDismissRequest,
+    ) {
+        AnimatedContent(
+            targetState = sheetType,
+            transitionSpec = {
+                when (targetState) {
+                    LedgerManualBottomSheetType.CREATE -> {
+                        slideInHorizontally { fullWidth -> fullWidth }
+                            .togetherWith(slideOutHorizontally { fullWidth -> fullWidth / -3 })
+                    }
+
+                    LedgerManualBottomSheetType.LIST -> {
+                        slideInHorizontally { fullWidth -> -fullWidth }
+                            .togetherWith(slideOutHorizontally { fullWidth -> fullWidth / 3 })
+                    }
+                }
+            }
+        ) { targetState ->
+            when (targetState) {
+                LedgerManualBottomSheetType.LIST -> {
+                    LedgerManualCategoryBottomSheetContent(
+                        categories = categories,
+                        onDismissRequest = onDismissRequest,
+                        onClickCreate = { sheetType = LedgerManualBottomSheetType.CREATE }
+                    )
+                }
+
+                LedgerManualBottomSheetType.CREATE -> {
+                    LedgerManualCategoryCreateBottomSheetContent(
+                        textFieldValue = categoryValue,
+                        isSystemCategoryError = isSystemCategoryError,
+                        categories = categories,
+                        onValueChange = onChangeCategoryValue,
+                        onClickRegister = {},
+                        onPrev = { sheetType = LedgerManualBottomSheetType.LIST }
+                    )
+                }
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+fun LedgerManualCategoryBottomSheetContent(
+    modifier: Modifier = Modifier,
+    categories: List<String>,
+    onDismissRequest: () -> Unit,
+    onClickCreate: () -> Unit,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(448.dp)
+            .background(White)
+            .padding(horizontal = MMHorizontalSpacing, vertical = 20.dp),
+    ) {
+        Icon(
+            modifier = Modifier
+                .align(alignment = Alignment.End)
+                .noRippleClickable(onDismissRequest),
+            painter = painterResource(R.drawable.ic_close_default),
+            contentDescription = null
+        )
+        Spacer(modifier = Modifier.height(4.dp))
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                style = Heading4,
+                color = Gray10,
+                text = "카테고리",
+            )
```

Comment:
디자인에서 텍스트 색상 `Black`으로 확인됩니다!

```suggestion
            Text(
                style = Heading4,
                color = Black,
                text = "카테고리",
            )
```